client/src/components/music-service-detector.tsx

```
@@ -0,0 +1,113 @@
+import React from 'react';
+import {
+  SiYoutubemusic,
+  SiSpotify,
+  SiApplemusic
+} from 'react-icons/si';
```

ESLint

[error] Replace `⏎··SiYoutubemusic,⏎··SiSpotify,⏎··SiApplemusic⏎` with `·SiYoutubemusic,·SiSpotify,·SiApplemusic·`

(prettier/prettier)